%YAML:1.0
transformation_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [  1.27755347e-02,   7.48846231e-02,  -2.97173802e-01, -2.46883581e-03,   1.59522560e-01,  -5.47704021e-01, -5.19536325e-06,   2.98962964e-04,   1.52353391e-02]
